game-core/src/main/java/games/strategy/engine/data/Route.java

```diff
@@ -160,18 +161,18 @@ public void add(final Territory territory) {
     if (territory == null) {
       throw new IllegalStateException("Null territory");
     }
-    if (territory.equals(m_start) || m_steps.contains(territory)) {
+    if (territory.equals(start) || steps.contains(territory)) {
       throw new IllegalArgumentException(String.format(
-          "Loops not allowed in m_routes, route: %s, new territory: %s", this, territory));
+          "Loops not allowed in steps, route: %s, new territory: %s", this, territory));
```

Comment:
What's m_routes?

Reply:
I guessed that it was the former name of the `m_steps` field, which is why I renamed it here.  It's the only thing that would make sense in this context (see the expression `steps.contains(...)` in the enclosing `if` statement).

Reply:
Found it: https://github.com/triplea-game/triplea/commit/93a3980d0ec1b4692d111413546da56f63dc20b2?diff=unified#diff-d0faf2d176e45d370c31ae9629b5b544R60

You are indeed correct. 👍